docs/processors.md

```diff
@@ -61,6 +62,58 @@ The resulting JSON string looks like this:
 {"type":"Person","first_name":"Homer","last_name":"Simpson","age":45}
 ```
 
+### `rfl::AddStructsToVariants` 
+
+This processor automatically adds structs to variants. Consider the following example:
```

Comment:
Suggestion: "automatically adds struct tags to"

Reply:
That was meant to be tags as well...